docs/src/LieTheory/intro.md

```diff
@@ -0,0 +1,31 @@
+```@meta
+CurrentModule = Oscar
+DocTestSetup = Oscar.doctestsetup()
+```
+
+```@meta
+CurrentModule = Oscar
+DocTestSetup = Oscar.doctestsetup()
+```
+
```

Comment:
```suggestion
```

Reply:
This seems to be duplicate

Reply:
oops. I had some issues with this file last night (somehow dropped some changes while rebasing) and this must have happened while rescuing the lost commit. Thanks for finding!